docs/modules/ROOT/pages/scheduling-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+= Configure scheduling policy for Hazelcast instances
```

Comment:
Good point. However, I think 'Configuring the Scheduler' is not a correct term either, because when you setup your Kubernetes environment, you 'configure the scheduler' by passing some program arguments to master. 
As far as I understand, the official docs use 'scheduling pods' for assigning pods to nodes, this is what we use here. 
What do you think about 'Configure Scheduling of Hazelcast Pods'?

Reply:
Or just 'Scheduling Hazelcast Pods'?